frontend/src/App.css

```diff
@@ -14,4 +14,11 @@
 
 .flex-fill {
     flex: 1 1 auto;
-}
\ No newline at end of file
+}
+
+.alert {
+    margin-bottom: 1px;
+    height: 50px;
+    line-height:50px;
+    padding:0px 15px;
+  }
```

Comment:
Notice this red circle with a '-' through it? It means no new line is present at the end of the file. The other files aren't missing it, you should add the new line and be consistent. Having a standard for this can be important if you're working with multiple people and they use different endings for files... it will show up as a diff whenever they save a file (if they strip or add). By default I've seen adding a new line as a behavior (but stripping it if developing on Windows).

Reply:
<img width="1113" alt="Screen Shot 2022-06-22 at 7 10 46 AM" src="https://user-images.githubusercontent.com/68614831/175050593-8e81c504-2307-4fb9-908c-6ec7f33736df.png">
Alignment

<img width="1113" alt="Screen Shot 2022-06-22 at 7 11 24 AM" src="https://user-images.githubusercontent.com/68614831/175050724-caf4f400-5248-4545-8d21-b3992defba54.png">
notification/alert